tests/foreman/cli/test_system_registration.py

```diff
@@ -7,96 +7,58 @@ class SystemRegistrationTestCase(CLITestCase):
     """Tests for System Registration CLI"""
 
     @stubbed()
-    def test_rpm_install(self):
-        """@test: rpm can be retrieved
+    def test_positive_register_with_no_ak(self):
+        """@test: Register Content host to satellite without activation key
 
-        @feature: system registration
+        @feature: Content host
 
-        @assert: rpm required for modifying rhsm.conf can be installed from sat
-        host
-
-        @assert: rhsm.conf has been updated appropriately.
-
-        @status: Manual
-        """
-
-    @stubbed()
-    def test_register_no_ak(self):
-        """@test: register system to sat without activation key
-
-        @feature: system registration
-
-        @assert: system successfully registered to appropriate org
-
-        @status: Manual
-        """
-
-    @stubbed()
-    def test_register_with_ak(self):
-        """@test: register system with activation key
-
-        @feature: system registration
-
-        @assert: system successfully registered to appropriate org and
-        subscribed to content as listed in key
-
-        @status: Manual
-        """
-
-    @stubbed()
-    def test_cannot_register_twice_negative(self):
-        """@test: Attempt to register a system twice to an instance
-
-        @feature: system registration
-
-        @assert: system cannot be registered twice via RHSM
+        @assert: Content host successfully registered to appropriate org
 
         @status: Manual
         """
 
     @stubbed()
-    def test_registered_systems_pull_content(self):
```

Comment:
this test is removed since test_contenthost.py already has tests which install package, package group, errata.